examples/open_pbr_velvet.mtlx

```diff
@@ -8,6 +8,6 @@
     <input name="specular_roughness" type="float" value="0.8" />
     <input name="fuzz_weight" type="float" value="1" />
     <input name="fuzz_color" type="color3" value="0.4, 0.4, 0.4" />
-    <input name="fuzz_roughness" type="float" value="0.2" />
+    <input name="fuzz_roughness" type="float" value="0.5" />

```

Comment:
Is this empirical, or is there a closed form to fit Conty-Kulla on Zeltner one ?

Reply:
I'm not aware of a closed-form solution, but we can definitely put together a good approximation and include it in a future shader translation graph from `standard_surface` to `open_pbr_surface`.

For now, I'm just estimating the visual equivalent of the original settings with Zeltner sheen, in order to roughly maintain the look of the example material in the new model.